src/map/packet_system.cpp

```diff
@@ -4152,6 +4152,10 @@ void SmallPacket0x096(map_session_data_t* const PSession, CCharEntity* const PCh
     // NOTE: This section is intended to be temporary to ensure that duping shenanigans aren't possible.
     // It should be replaced by something more robust or more stateful as soon as is reasonable
     CCharEntity* PTarget = (CCharEntity*)PChar->GetEntity(PChar->TradePending.targid, TYPE_PC);
+    if (!PTarget)
+    {
+        return;
+    }
 
     // Clear pending trades on synthesis start
     if (PTarget && PChar->TradePending.id == PTarget->id)
```

Comment:
further checks that it exists after your addition are now no longer needed

Reply:
```cpp
    if (PChar->UContainer->GetType() != UCONTAINER_EMPTY)
    {
        if (PTarget)
        {
           
```
here too

Reply:
Indeed it can. 👍